cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -86,6 +86,19 @@ def custom_categories(appliance):
     category.delete_if_exists()
 
 
+def create_dialog(appliance, element_data, label=None):
+    service_dialog = appliance.collections.service_dialogs
+    if not label:
+        label = fauxfactory.gen_alphanumeric(15, start="label_")
+    sd = service_dialog.create(label=label, description="my dialog")
+    tab = sd.tabs.create(tab_label=fauxfactory.gen_alphanumeric(start="tab_"),
+                         tab_desc="my tab desc")
+    box = tab.boxes.create(box_label=fauxfactory.gen_alphanumeric(start="box_"),
+                           box_desc="my box desc")
+    element = box.elements.create(element_data=[element_data])
+    return sd, element
+
```

Comment:
Instead of this function. please check if you can use this fixture in this file: https://github.com/ManageIQ/integration_tests/pull/9995/files#diff-faeac2d3d489ac01e0ebae6983c9c3cbR36

Reply:
Actually I do not want to pass same input to `create_dialog` so kept it as a function instead of fixture

Reply:
- you are already passing input ```dialog, element = create_dialog(appliance, element_data)```. 
- Also, for now, I can see this function used for this test case only, Want to know use of **label=None** argument here? or we can directly provide ```sd = service_dialog.create(label=fauxfactory.gen_alphanumeric(15, start="label_"), description="my dialog")```

Reply:
I have updated the function signature now